http-client/src/test/java/io/airlift/http/client/AbstractHttpClientTest.java

```diff
@@ -181,11 +181,12 @@ public void abstractTeardown()
         }
     }
 
-    @Test(enabled = false, description = "This takes over a minute to run")
```

Comment:
This test takes ~45minutes in CI, but only 10s for each of the 6 sub classes on my laptop, so I think we should disable this before merging.

Reply:
Are you saying CI is 45x slower?

does it make sense to run the test for at least one of the subclasses?

Reply:
Yes maybe.  I think the only important one is plain old http/1.  We could also lower the number to like 30k to make the test faster.